Ref.Windows/Export/BibTexExporter.cs

```diff
@@ -96,6 +98,29 @@ internal void WriteBook(TextWriter writer, Book book)
             WriteEntry(writer, "@book", book.Key, fields);
         }
 
+        internal void WriteThesis(TextWriter writer, Thesis thesis)
+        {
+            if (IsUnkeyed(thesis.Key, thesis.Title, writer))
+                return;
+
+            var fields = new List<string>();
+            AddField(fields, ReplaceSemicolonsWithAnd(thesis.Author), "author");
+            AddField(fields, thesis.Doi, "doi");
+            AddField(fields, thesis.Isbn, "isbn");
+
+            // Licentiate thesis is a special case of PhD
+            // TODO: Hardcoded string
+            if (thesis.Kind == ThesisKind.Licentiate)
+                AddField(fields, "Licentiate thesis", "type");
```

Comment:
If there are new thesis types added, this must be updated as well. I think silently generating slightly incorrect data is better than crashing in this case.